update point about tab navigation (#815)
mate-academy/layout_miami: checklist.md

    });
    ```
16. Remember to reset default margins for headings (h1, h2, h3, etc.). Make sure that ```<h1>``` is positioned exactly 80px from the bottom of the header.

> 17. Make sure that on navigate to the bottom of your page with `TAB` key while the menu is open your page blocks don't visible over the menu like on screen:

Reviewer: ```suggestion
17. Make sure that when you navigate to the bottom of your page with `TAB` key, while the menu is open, gallery pictures are not visible over the menu, example:
```
